scripts/prepConsent.js

```diff
@@ -0,0 +1,67 @@
+const fs = require('fs');
+const path = require('path');
+const syncCsvParse = require('csv-parse/lib/sync');
+const _ = require('lodash');
+const moment = require('moment');
+
+
+// Check command line arguments
+if (process.argv.length !== 3) {
+  console.log('Command line argument missing, aborting...');
+  console.log('');
+  console.log('Usage: node tools/consent-teacher-moments.js tmp/PRIVATE_SENSITIVE_DATA/consent-teacher-moments/UserConsent.csv');
```

Comment:
Script looks good!  This bit looks stale, or you could just remove this and put it in the README.

Reply:
Oh right. I updated just a small part of it and forgot to check the rest of the command.

I'm planning on having this bit be in both the README and in this code - similar to how things were in the private repo